compiler/il/VectorOperations.enum

```diff
@@ -1463,6 +1695,183 @@ VECTOR_OPERATION_MACRO(\
    /* .ifCompareOpCode         = */ TR::BadILOp, \
    /* .description             =    get first non-zero vector element masked */ \
 )
+// GITA
+VECTOR_OPERATION_MACRO(\
+   /* .operation               = */ vmpopcnt, \
+   /* .name                    = */ "vmpopcnt", \
+   /* .properties1             = */ 0, \
+   /* .properties2             = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE, \
+   /* .properties3             = */ ILProp3::LikeUse, \
+   /* .properties4             = */ 0, \
+   /* .dataType                = */ TR::NoType, \
+   /* .typeProperties          = */ ILTypeProp::VectorResult, \
+   /* .childProperties         = */ ONE_CHILD(ILChildProp::UnspecifiedChildType), \
+   /* .swapChildrenOperation   = */ TR::vBadOperation, \
+   /* .reverseBranchOperation  = */ TR::vBadOperation, \
+   /* .booleanCompareOpCode    = */ TR::BadILOp, \
+   /* .ifCompareOpCode         = */ TR::BadILOp, \
+   /* .description             =    vector lanewise bit count with mask*/ \
```

Comment:
For consistency with the other descriptions and for readability, can you add a space before the trailing `*/` please?